cdc/log.cc

```diff
@@ -962,6 +962,14 @@ class transformer final : public change_processor {
         _tuuid = timeuuid_type->decompose(generate_timeuuid(ts));
     }
 
+    void produce_preimage(const clustering_key* ck, const one_kind_column_set& columns_to_include) override {
+        // TODO
```

Comment:
I think this patch should be embedded in the following patch that actually makes use of those methods.

Reply:
Done